src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -223,73 +249,129 @@ public Task<ImmutableArray<CodeActionOperation>> GetPreviewOperationsAsync(Cance
         }
 
         /// <summary>
-        /// Override this method if you want to implement a <see cref="CodeAction"/> subclass that includes custom <see cref="CodeActionOperation"/>'s.
+        /// Override this method if you want to implement a <see cref="CodeAction"/> subclass that includes custom <see
+        /// cref="CodeActionOperation"/>'s.
         /// </summary>
         protected virtual async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(CancellationToken cancellationToken)
```

Comment:
Consider ban this method in Roslyn repo

Reply:
same for other twin methods not taking an IProgress

Reply:
sounds good.

Reply:
done.